dqm-root/src/common/VFAT_histogram.cxx

```diff
@@ -27,8 +27,8 @@ class VFAT_histogram: public Hardware_histogram
     Header     = new TH1F("Header", "Header", 32,  0x0 , 0xff);
     SlotN    = new TH1F("SlotN", "Slot Number", 24,  0, 24);
     FiredChannels = new TH1F("FiredChannels", "FiredChannels", 128, -0.5, 127.5);
-    //FiredStrips   = new TH1F("FiredStrips",   "FiredStrips",   128, -0.5, 127.5);
     latencyScan   = new TH1F("latencyScan",   "Latency Scan", 256,  -0.5, 255.5);
+    latencyScan2D = new TH2F("latencyScan2D", "Latency Scan: Chan Vs Latency", 1024, -0.5, 1023.5, NCHANNELS, -0.5, NCHANNELS-0.5);
```

Comment:
Are we ever going to scan 0-1024 on the cosmic stand? If not, I would reduce the upper latency border to 255.5 If yes - please also increase the range of 1D latencyScan histogram. Otherwise fine.

Reply:
We might; especially when working on the Coffin setup in conjunction with the CSC; this was the case at the 904 Camp last year.  And yeah I'll update the 1D distribution.

Reply:
well, a special tag can be released for coffin, (two chamber setup vs 15 chambers/3 AMCs setup, which is virtual 36 chambers setup!!!), but ok for the moment... Hope DPG will install the online DQM soon and we stop misuse of the expert tools

Reply:
Indeed